ide/che-core-ide-stacks/src/main/resources/images/type-bitnami.svg

```diff
@@ -0,0 +1,177 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
```

Comment:
This icon is missing a licence header. 

You can give a look at this one:
https://github.com/eclipse/che/blob/master/ide/che-core-ide-stacks/src/main/resources/images/type-che.svg?short_path=2182d77#L2-L13